org.multimc.MultiMC.appdata.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<component type="desktop">
+  <id>org.multimc.MultiMC</id>
+  <launchable type="desktop-id">org.multimc.MultiMC.desktop</launchable>
+  <name>MultiMC</name>
+  <summary>Manage Minecraft instances with ease</summary>
+  <description>
+    <p>Overview</p>
+    <p>MultiMC is a free, open source launcher for Minecraft. It allows you to have multiple, cleanly separated instances of Minecraft (each with their own mods, texture packs, saves, etc) and helps you manage them and their associated options with a simple and powerful interface.</p>
+    <p>Features</p>
+    <p>Manage multiple instances of Minecraft at once</p>
+    <p>Start Minecraft with a custom resolution</p>
+    <p>Change Java's runtime options (including memory options)</p>
+    <p>Shows Minecraft's console output in a colour coded window</p>
+    <p>Kill Minecraft easily if it crashes / freezes</p>
+    <p>Custom icons and groups for instances</p>
+    <p>Forge integration (automatic installation, version downloads, mod management)</p>
+    <p>Minecraft world management</p>
+    <p>Import and export Minecraft instances to share them with anyone</p>
+    <p>Supports every version of Minecraft that the vanilla launcher does</p>
+    <p>NOTE: This distribution is not verified by, affiliated with, or supported by the MultiMC team.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://multimc.org/images/screenshots/main.png</image>
+      <image type="source">https://multimc.org/images/screenshots/editmods.png</image>
+      <image type="source">https://multimc.org/images/screenshots/version.png</image>
+      <image type="source">https://multimc.org/images/screenshots/console.png</image>
+      <image type="source">https://multimc.org/images/screenshots/settings.png</image>
+    </screenshot>
+  </screenshots>
+  <releases>
+    <release version="0.6.11" date="2020-3-28" />
+  </releases>
+  <url type="homepage">https://multimc.org/</url>
+  <url type="help">https://discord.com/invite/0k2zsXGNHs0fE4Wm</url>
+  <url type="faq">https://github.com/MultiMC/MultiMC5/wiki/FAQ</url>
+  <url type="bugtracker">https://github.com/flathub/org.multimc.MultiMC/issues</url>
+  <url type="translate">https://translate.multimc.org/</url>
+  <url type="donation">https://www.patreon.com/multimc</url>
+  <developer_name>The MultiMC Team</developer_name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>Apache License 2.0</project_license>
```

Comment:
the proper SPDX ID is "Apache-2.0"

source https://spdx.org/licenses/

(hence the failure)

Reply:
Thank you, I'll try rebuilding with this fix!